content/jobs/README.md

```diff
@@ -0,0 +1,46 @@
+# Job Post Instructions
+
+---
+
+Hey there! If you're a local-ish tech company looking to hire talent we at NewHaven.io would love to help out by putting some information about your organization and open role(s) on our website.
+
+Just clone this repository, add your information to [this page](https://github.com/newhavenio/newhavenio/tree/master/content/jobs/jobs.yaml), and submit a pull request.
+
+```markdown
+- company: "NewHaven.IO"
+  jobURL: "https://newhaven.io"
+  jobTitle: "Community Moderator"
+  location: "New Haven, CT // Remote"
```

Comment:
Can we have a "company location" and a "remote friendly" field?

Reply:
Pushing a potential solution for the remote friendly field now.

Reply:
Cool. My take here is that we should force companies to say where they are but also say if remote is ok.

Reply:
That's the change I pushed. :)

Reply:
Definitely trying to make as much information required as possible so people don't just spam our members.